SI/Sem.04/Pract.04/Task 06.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+
+bool isCapitalLetter(char S)
+{
+    if (S >= 'A' && S <= 'Z')
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
+
+bool checkSymbol(char S)
+{
+    if ((S >= 'A' && S <= 'Z') || (S>='a' && S<='z'))
```

Comment:
(S >= 'A' && S <= 'Z') || (S>='a' && S<='z') is a boolean expression, so it returns a boolean value that you can directly return